components/centraldashboard/public/components/main-page.js

```diff
@@ -173,11 +173,16 @@ export class MainPage extends utilitiesMixin(PolymerElement) {
      * @param {Event} ev AJAX-response
      */
     _onHasWorkgroupResponse(ev) {
-        const {user, hasWorkgroup, hasAuth} = ev.detail.response;
+        const {user, hasWorkgroup, hasAuth, 
+            registrationFlowAllowed} = ev.detail.response;
         this._setIsolationMode(hasAuth ? 'multi-user' : 'single-user');
-        if (hasAuth && !hasWorkgroup) {
+        if (hasAuth) {
             this.user = user;
-            this._setRegistrationFlow(true);
+            if (registrationFlowAllowed && !hasWorkgroup) {
+                this._setRegistrationFlow(true);
+            } else {
+                this._setRegistrationFlow(false);
+            }
         }
```

Comment:
Delete this, I made a mistake in my last review it should be this instead
```suggestion
        if (registrationFlowAllowed && hasAuth && !hasWorkgroup) {
            this.user = user;
            this._setRegistrationFlow(true);
        }
```

Reply:
Also Luis, could you please sign the CLA or follow up on the steps from @googlebot:

> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fkubeflow%2Fkubeflow%2Fpull%2F5080) for more info**.

Reply:
@googlebot I signed it!